### ISSUE TITLE
Node uptime and node graph

### ISSUE THREAD

**Issue by Oxim0v** [pterodactyl/panel#4378]
### Is there an existing feature request for this?

- [x] I have searched the existing issues before opening this feature request.

### Describe the feature you would like to see.

In the admin panel I think it would be good to display the uptime of a node and to be able to schedule a task (such as rebooting the node in the same principle as for servers)
And also displayed the average uptime of all the nodes combined and put a graph in the page of the node for the use of the processor and the ram.

### Describe the solution you'd like.

For example for the use of the processor and the ram (global) a graph as for the servers (see the screen below)

![Capture d’écran 2022-09-15 155614](https://user-images.githubusercontent.com/53126647/190422866-5a8a67b0-531b-4c9d-afd3-876fb3af0a89.png)

and for the uptime of the node add it in the information section
![Capture d’écran 2022-09-15 155959](https://user-images.githubusercontent.com/53126647/190423650-3639e9f8-90a4-4568-b215-2762aa4f1e73.png)

**Comment by richiedevs** [comment]
Maybe this should be an optional features

**Comment by argonaut-network** [comment]
I believe Wings pulls that information from each Docker container's stats using the Docker socket. Problematically, the Docker socket does not provide a way to monitor system-wide resource usage, so the team would be required to engineer a brand new solution -- or at least use a different one than what's used now -- to accomplish this.

**Comment by matthewpi** [comment]
> I believe Wings pulls that information from each Docker container's stats using the Docker socket. Problematically, the Docker socket does not provide a way to monitor system-wide resource usage, so the team would be required to engineer a brand new solution -- or at least use a different one than what's used now -- to accomplish this.

Wings can do things outside of the Docker socket which is shown by half the functionality that exists in it.  Please don't state information about our software architecture without actually looking into it.

---

In terms of the issue, I can see showing an uptime for nodes individually (both the system itself and wings independently), but historical uptime or graphs, especially in aggregate will not be added.

**Comment by argonaut-network** [comment]
Fair enough. I assumed Wings used the Docker socket for monitoring resource usage of containers specifically, since the containerized version of Wings doesn't ask for access to /proc. You know what they say about assumptions, though.

**Comment by DaneEveritt** [comment]
In the interest of maintainer sanity and cleaning up an enormous backlog of stale feature requests we have moved to requiring all feature requests be discussed within [Discussions](https://github.com/orgs/pterodactyl/discussions/categories/feature-requests). Please consider re-opening this request there if you believe it is still relevant and worth consideration.

See https://github.com/pterodactyl/panel/issues/5491 for additional context.

Thanks.

<sub>_This comment is in the interest of issue tracker maintenance: it does not imply that maintainers are endorsing or dismissing your request._</sub>
